### PR TITLE
Adds delete & update feature to admin/system-roles route

### LINKED ISSUE #766: Implement roles API for fetching roles in admin/system-roles
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Work to be done:
- Fetch all the roles and list it
- Implement delete & update role feature

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds delete & update feature to admin/system-roles route

Fixes #766

### CODE REVIEW

**Comment by geekyd** [comment]
@CosmicCoder96 @sumedh123 @harshitagupta30 The dropdown while updating does not seem to work, I tried different things but did not succeed. If you know how to fix it please help. Thanks

**Comment by niranjan94** [comment]
Deployment link: https://pr-772-open-event-frontend-fossasia.surge.sh

File: app/templates/admin/permissions/system-roles.hbs [hunk 1]
```diff
@@ -1,121 +1,109 @@
...
             </div>
           </td>
-          <td>{{ui-checkbox label=(t 'Unverified User') checked=userPermission.unverifiedUser onChange=(action (mut userPermission.unverifiedUser))}}</td>
           <td>
-            {{ui-checkbox label=(t 'Anonymous User') checked=userPermission.anonymousUser onChange=(action (mut userPermission.anonymousUser))}}
+            <div class="ui horizontal compact basic buttons">
+              {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action 'openAddSystemRoleModal' role)}}
+                <i class="edit icon"></i>
+              {{/ui-popup}}
+              {{#ui-popup content=(t 'Delete') class='ui icon button' click=(action (confirm (t 'Are you sure you would like to delete this page?') (action 'deleteSystemRole' role)))}}
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions/system-roles.hbs]: Please change the confirmation message to `role` instead of `page`

**Comment by niranjan94** [comment]
Deployment link: https://pr-772-open-event-frontend-fossasia.surge.sh

**Pull request merged**